tests/test_message_request_handler.py

```diff
@@ -17,6 +17,6 @@ def test_show_popup(self):
                 {'title': "def"}
             ]
         }
-        handler = MessageRequestHandler(view, session, "1", params, 'lsp server')
-        handler.show()
-        self.assertTrue(view.is_popup_visible())
+        manager = WindowManager(window, None, None)
+        manager.handle_message_request(session, params, "1")
+        self.assertEquals(view.element(), "input:input")
```

Comment:
`view.element()` is always returning None. I am not sure what is wrong here

Reply:
Looks like `Window.active_view()` always returns the active editor view (= tab), even if the input focus is in a widget / input field. Not sure whether it is intentional or a bug.

Reply:
You probably need to remove all references to the removed notifications.css since it's currently breaking the popups.

Reply:
No idea if related to this thread actually but the popups are broken. :)

Reply:
> You probably need to remove all references to the removed notifications.css since it's currently breaking the popups.

I just did but same problem

Reply:
> No idea if related to this thread actually but the popups are broken. :)

After https://github.com/sublimelsp/LSP/pull/1915/commits/405cd7b4899f83abde93772384d18100d9116b68 they should be unbroken now